grapher/Models/Theming/IO/ThemeFileOperations.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace grapher.Models.Theming.IO
+{
+    public class ThemeFileOperations
+    {
+        public string ThemePath { get; set; }
+
+        public IEnumerable<ColorScheme> LoadThemes()
+        {
+            ThemePath = Path.Combine(Environment.CurrentDirectory, "/themes");
```

Comment:
Don't need the unix path character /

Reply:
ThemePath = Path.Combine(Environment.CurrentDirectory, "themes");

Reply:
Has been resolved in latest commit